lib/keystores.js

```diff
@@ -8,12 +8,14 @@ const bedrock = require('bedrock');
 const database = require('bedrock-mongodb');
 const {LruCache} = require('@digitalbazaar/lru-memoize');
 const {BedrockError} = bedrock.util;
+const pLimit = require('p-limit');
 
 // load config defaults
 require('./config');
 
 const USAGE_COUNTER_MAX_CONCURRENCY = 100;
 let KEYSTORE_CONFIG_CACHE;
+const limit = pLimit(USAGE_COUNTER_MAX_CONCURRENCY);
```

Comment:
This would limit across calls to `getStorageUsage` instead of within them. To keep things as they were, `limit` should be moved below.

```suggestion
const USAGE_COUNTER_MAX_CONCURRENCY = 100;
let KEYSTORE_CONFIG_CACHE;
```